src/service/real-time-config/real-time-config-impl.js

```diff
@@ -502,45 +515,67 @@ export class RealTimeConfigManager {
     timeoutMillis,
     callout,
     checkStillCurrent,
-    errorReportingUrl
+    errorReportingUrl,
+    element
   ) {
+    let rtcFetch;
+    if (isAmpScriptUri(url)) {
+      rtcFetch = Services.scriptForDocOrNull(element)
+        .then((service) => {
+          userAssert(service, 'AMP-SCRIPT is not installed.');
+          return service.fetch(url);
+        })
+        .then((json) => {
+          checkStillCurrent();
+          const rtcTime = Date.now() - this.rtcStartTime_;
+          if (typeof json !== 'object') {
```

Comment:
Do we need to try to parse?

Reply:
Nope! I know I just named it `fetch`, but it isn't actually that much like the browser's fetch. No need to call `.text()` or `.json()` on the response. It essentially received a `postMessage`'d response, which is already JSON (or a primitive in which case we throw)